cmd/devtools/helpers.go

```diff
@@ -0,0 +1,37 @@
+package devtools
+
+import (
+	"fmt"
+	"os"
+	"regexp"
+
+	log "github.com/sirupsen/logrus"
+)
+
+func IsInstanceNameValidOrPanic(instance string) {
+	re, err := regexp.Compile(`^[a-z]+[a-z0-9]*(-[a-z0-9]+)*$`)
+	if err != nil {
+		log.WithError(err).Error("Error compiling regex")
+		panic(err)
+	}
+	if !re.MatchString(instance) {
+		log.Errorf("Invalid instance name: %s", instance)
+		err := fmt.Errorf(`
+Invalid instance name: '%s'. Instance names must be lowercase, alphanumeric, 
+and may contain dashes. It can't begin or end with a dash. No repeating dashes.
+`, instance)
+		panic(err)
+	}
+}
+
+// CreateDir creates a directory with the given name with 0755 permissions.
+// If the directory can't be created, it will panic.
+func CreateDir(dirName string) {
+	// TODO - add error handling?
+	err := os.MkdirAll(dirName, 0755)
+	if err != nil {
+		log.WithError(err).Error("Error creating data directory")
+		panic(err)
```

Comment:
I'm on the fence about panicking here. It's not validating user input like the other function, but it does make our code cleaner.

I think if we panic here we should name the function `CreateDirOrPanic`.
Can remove the `TODO`!

Reply:
I changed the name to `CreateDirOrPanic`. I think panicing here is okay because if the cli can't create the directories nothing else is going to run. But probably worth taking another look at again in the future.